docs/how to/configure-rvs.md

```diff
@@ -0,0 +1,140 @@
+are keyed lists themselves (nested mappings).
+
+Action item (or action for short) uses keys to define nature of validation test to be performed. Each action has some common keys -- like 'name', 'module', 'deviceid' -- and test specific keys which depend on the module being used.
+
+An example of a RVS configuration file is given here:
+
+```
+    actions:
+    - name: action_1
+      device: all
+      module: gpup
+      properties:
+        mem_banks_count:
+      io_links-properties:
+        version_major:
+    - name: action_2
+      module: gpup
+      device: all
+      properties:
+        mem_banks_count:
+    - name: action_3
+    ...
+```
+
+## Common Configuration Keys
+
+Common configuration keys applicable to most modules are summarized in the following table.
+
+<table>
+<tr><th>Config Key</th> <th>Type</th><th> Description</th></tr>
+<tr><td>name</td><td>String</td><td>The name of the defined action.</td></tr>
+<tr><td>device</td><td>Collection of String</td>
+<td>This is a list of device indexes (gpu ids), or the keyword “all”. The
+defined actions will be executed on the specified device, as long as the action
+targets a device specifically (some are platform actions). If an invalid device
+id value or no value is specified the tool will report that the device was not
+found and terminate execution, returning an error regarding the configuration
+file.</td></tr>
+
+<tr><td>deviceid</td><td>Integer</td><td>This is an optional parameter, but if
+specified it restricts the action to a specific device type
+corresponding to the deviceid.</td></tr>
+<tr><td>parallel</td><td>Bool</td><td>If this key is false, actions will be run
+on one device at a time, in the order specified in the device list, or the
+natural ordering if the device value is “all”. If this parameter is true,
+actions will be run on all specified devices in parallel. If a value isn’t
+specified the default value is false.</td></tr>
+
+<tr><td>count</td><td>Integer</td><td>This specifies number of times to execute
+the action. If the value is 0, execution will continue indefinitely. If a value
+isn’t specified the default is 1. Some modules will ignore this
+parameter.</td></tr>
+
+<tr><td>wait</td><td>Integer</td><td>This indicates how long the test should
+wait
+between executions, in milliseconds. Some
+modules will ignore this parameter. If the
+count key is not specified, this key is ignored.
+duration Integer This parameter overrides the count key, if
+specified. This indicates how long the test
+should run, given in milliseconds. Some
+modules will ignore this parameter.</td></tr>
+
+<tr><td>module</td><td>String</td><td>This parameter specifies the module that
+will be used in the execution of the action. Each module has a set of sub-tests
+or sub-actions that can be configured based on its specific
+parameters.</td></tr>
+</table>
+
+### Command Line Options
```

Comment:
Same issues with this table

Reply:
This is the original Markdown file, @lpaoletti. We decided to keep the original file format for now given the time constraint.